rosidl_runtime_cpp/include/rosidl_runtime_cpp/traits.hpp

```diff
@@ -129,14 +129,20 @@ inline void value_to_yaml(const std::string & value, std::ostream & out)
 inline void value_to_yaml(const std::u16string & value, std::ostream & out)
 {
   out << "\"";
+#if __cplusplus < 201703L
```

Comment:
i think we could use this pre-processor to handle this situation across C++ standards, there are `msg__struct.hpp.em` already uses this kind of work-around .